opentreemap/treemap/instance.py

```diff
@@ -112,6 +113,12 @@ def get_instance_permission_spec():
                              'of a tree or map feature'),
             'default_role_names': [Role.ADMINISTRATOR, Role.EDITOR],
             'label': _('Can View External Link')
+        },
+        {
+            'codename': PERMISSION_ACCESS_WORKS_MANAGEMENT,
+            'description': _('Can view or modify Works Management entities'),
+            'default_role_names': [Role.ADMINISTRATOR],
+            'label': _('Can Access Works Management')
```

Comment:
Sorry if I wasn't clear. Just summarizing the correct behavior I was seeing. All good.